Library/Formula/cv.rb

```diff
@@ -0,0 +1,20 @@
+class Cv < Formula
+  desc "Coreutils Viewer: Show progress for coreutils"
+  homepage "https://github.com/Xfennec/cv"
+  url "https://github.com/Xfennec/cv/archive/v0.6.tar.gz"
+  sha256 "4291ab2f0c507df9eb694bf956ed65dc12bae3c492b795235142b5b87e2e2239"
+  head "https://github.com/Xfennec/cv.git"
+
+  patch do
+    url "https://patch-diff.githubusercontent.com/raw/Xfennec/cv/pull/19.diff"
```

Comment:
Mind making this `https://github.com/Xfennec/cv/pull/19.patch`? It just makes it easier to jump to the actual PR.